packages/manager/modules/account-migration/src/notification/index.js

```diff
@@ -0,0 +1,36 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import moment from 'moment';
+
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ui-kit';
+
+import accountMigrationNotification from './notification.component';
+
+const moduleName = 'accountMigrationNotification';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
+    'oui',
+    'pascalprecht.translate',
+    translate,
+  ])
+  .component('accountMigrationNotification', accountMigrationNotification)
+  .run(
+    /* @ngInject */ ($translate) => {
+      let lang = $translate.use();
+
+      if (['en_GB', 'es_US', 'fr_CA'].includes(lang)) {
+        lang = lang.toLowerCase().replace('_', '-');
+      } else {
+        [lang] = lang.split('_');
+      }
+      return import(`script-loader!moment/locale/${lang}.js`).then(() =>
+        moment.locale(lang),
+      );
+    },
+  )
```

Comment:
`moment` locale is already being configured in manager dedicated app. But still, it gives the date in english always. This is  fixed by passing on the expected locale in the place where we need the formatted date (`moment(migrationDates.START, 'MM/DD/YYYY', lang).format('LL')`), so that we are not change any global settings. Thanks!